srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/EditRiskModelController.java

```diff
@@ -39,51 +57,113 @@ public EditRiskModelController(final AdminService adminService )
         fAdminService = adminService;
     }
     
-    @ModelAttribute(ATTRIBUTE_RISK_MODEL)
-    public EditRiskModel createEditRiskModel(
-            @PathVariable final int riskModelId )
-            throws InvalidIdentifierException
+    /**
+     * <p>Returns a ModelAndView for editing the given EditRiskModel.</p>
+     * 
+     * <p>NOTE: Not a request handler method.
+     */
+    public ModelAndView displayForm(final EditRiskModel editModel)
     {
-        RiskModel rm = fAdminService.getRiskModelForId(riskModelId);
-        
+        final ModelAndView mv = new ModelAndView( Views.EDIT_RISK_MODEL );
+        mv.addObject(ATTRIBUTE_RISK_MODEL, editModel);
+        mv.addObject(ATTRIBUTE_SUMMARIES, editModel.makeTermSummaries(fAdminService));
+        return mv;
+    }
+
+    /**
+     * Displays a page for editing the given existing RiskModel.
+     * @param riskModelId the ID (surrogate key) of the RiskModel to edit
+     * @throws InvalidIdentifierException if no RiskModel exists with the ID
+     */
+    @RequestMapping(method = RequestMethod.GET)
+    public ModelAndView displayForm(
+            @PathVariable final int riskModelId ) throws InvalidIdentifierException
+    {
+        final RiskModel rm = fAdminService.getRiskModelForId(riskModelId);
```

Comment:
I don't think we would need to make this change, but a lot of websites would probably give a 404 error in this case and others when the URL is bad. We probably don't have enough time to worry about that, but do you think it's worth investigating?